src/main/java/se/idsec/signservice/integration/rest/SignServiceIntegrationApplication.java

```diff
@@ -86,16 +84,7 @@ CommonsRequestLoggingFilter requestLoggingFilter() {
     return loggingFilter;
   }
 
-  /**
-   * To allow Spring Boot actuator's httptrace.
-   * 
-   * @return a trace repository
-   */
-  @Bean
-  HttpTraceRepository htttpTraceRepository() {
-    return new InMemoryHttpTraceRepository();
-  }
-  
+
   /**
```

Comment:
Note that HttpTraceRepository was removed from Actuator in Spring Boot 3. This had to be removed.